commandapi-core/src/main/java/dev/jorel/commandapi/executors/CommandArguments.java

```diff
@@ -179,6 +191,111 @@ public Optional<Object> getOptional(String nodeName) {
 		}
 		return Optional.of(argsMap.get(nodeName));
 	}
+
+	/**
+	 * Returns a raw argument by its position
+	 *
+	 * @param index The position of this argument
+	 * @return An argument which is placed at the given index, or {@code null} if the provided index does not point to an argument.
+	 */
+	@Nullable
+	public String getRaw(int index) {
+		if (rawArgs.length <= index) {
+			return null;
+		} else {
+			return rawArgs[index];
+		}
+	}
+
+	/**
+	 * Returns a raw argument by its node name
+	 *
+	 * @param nodeName The node name of this argument. This was set when initializing an argument
+	 * @return A raw argument which has the given node name. Can be null if <code>nodeName</code> was not found.
```

Comment:
This description is inconsistent with the `getRaw` method that uses `{@code null}` in it. It would probably be better to use `{@code null}` here as well.